src/docs/product/performance/filters-display.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: "Filters & Display"
+sidebar_order: 20
+redirect_from:
+  - /product/performance/display/
+description: "Manage the information on the Performance page using search or page-level display filters to quickly identify performance issues."
+---
+
+You can filter the information displayed on the **Performance** page by searching or using page-level display filters.
+
+## Transaction and Trend Views
+
+The page default is to view transactions, but you can use the “By Transaction/By Trend” toggle to change the overall display of the page. The different ways you can filter and set the display of transactions is described fully on this page. To learn more about viewing transactions by trend, see our [Trends View documentation](/product/performance/trends/).
+
+## Search Condition Filters
+
+The search bar operates similarly to the one in the Discover [Query Builder](/product/discover-queries/query-builder/#filter-by-search-conditions). For example, you can refine your transaction search to a specific release by using `release:` as a key field and assigning the version as a value.
+
+## Display Filter
+
+The "Display" filter is next to the search bar. It automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, though note that some modes are not applicable to the platform.
```

Comment:
Could also be `not applicable for the selected platform`. The current wording just feels off.
```suggestion
The "Display" filter is next to the search bar. It automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, though note that some modes are not applicable to all platforms.
```